### ISSUE TITLE
Imagenet example shell script is only for validation

### ISSUE THREAD

**Issue by MaxVanDijck** [pytorch/examples#933]
https://github.com/pytorch/examples/tree/master/imagenet

The shell script specified in the readme only sorts the images of the validation set into folders
There is no script to extract the training set included in the readme
I suggest including this https://gist.github.com/BIGBALLON/8a71d225eff18d88e469e6ea9b39cef4 in the readme to extract and prep both the training and validation .tar files

**Comment by msaroufim** [comment]
Hi @MaxVanDijck would you like to make a PR for this?

**Comment by MaxVanDijck** [comment]
Yes, I can update the link in the readme today

**Comment by MaxVanDijck** [comment]
PR addressing issue: https://github.com/pytorch/examples/pull/977
